gdpr_assist/models.py

```diff
@@ -292,3 +609,13 @@ def get_target(self):
         except model.DoesNotExist:
             return None
         return obj
+
+    def summary(self):
```

Comment:
Can we just make that `__str__` instead? Or do you think the string is too long for that?